yarn.lock

```diff
@@ -3157,7 +3157,7 @@ diff@^1.3.2:
   version "1.4.0"
   resolved "https://registry.yarnpkg.com/diff/-/diff-1.4.0.tgz#7f28d2eb9ee7b15a97efd89ce63dcfdaa3ccbabf"
 
-diff@^3.2.0, diff@^3.3.1, diff@^3.5.0:
+diff@^3.3.1, diff@^3.5.0:
```

Comment:
Do you know where these changes come from?

Reply:
Hm, sadly this is also unclear to me :thinking: